tests/integration_tests.py

```diff
@@ -0,0 +1,49 @@
+import requests
+
+
+class PetStore:
+    """Petstore class.
```

Comment:
Increased this to comply with Google style docstrings. Generally we didn't follow this for tests, but since you had already added a docstring, I thought I might as well...